artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ActiveMQServerImpl.java

```diff
@@ -4586,9 +4586,12 @@ private void deployReloadableConfigFromConfiguration() throws Exception {
          for (BridgeConfiguration newBridgeConfig : configuration.getBridgeConfigurations()) {
             newBridgeConfig.setParentName(newBridgeConfig.getName());
             Bridge existingBridge = clusterManager.getBridges().get(newBridgeConfig.getParentName());
+            if (existingBridge == null) {
+               existingBridge = clusterManager.getBridges().get(newBridgeConfig.getParentName() + "-0");
```

Comment:
I don't understand what is this concatenation here. is this really necessary?

Reply:
It doesn't seem very "product" to me.. it would work somewhere individually.. but it does not seem right to me.

Reply:
@clebertsuconic 
I agree, that really wasn't nice. I was using the concatenation there because concurrent bridges get named uniquely to be manageable by appending -0, -1 and so on to their name. At this point that has yet to happen to the newBridge.

I made some changes here and also rebased against main, hope it's looking better.